GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -1313,6 +1317,8 @@ private void StageClick(object sender, EventArgs e)
 
         private void StagedFileContext_Opening(object sender, System.ComponentModel.CancelEventArgs e)
         {
+            //Do not show if no item selected
+            e.Cancel = !Staged.SelectedItems.Any();
```

Comment:
should it return at this point, if cancelled?

Reply:
It could, but it does not matter as the menu is not shown anyway.
I opted to skip the code

Reply:
technically it does, since we would be unnecessary executing code

Reply:
Yes, executing code in a case that normally not occurs anyway, a change will add code for the normal situation.
I changed the code to add a check (the alternative would have been an additional comment).

Even if I disagree: Thanks for the comments.